rollup.config.js

```diff
@@ -1,53 +1,60 @@
-import vue from 'rollup-plugin-vue' // Handle .vue SFC files
+import vue from 'rollup-plugin-vue'
 import babel from 'rollup-plugin-babel'
-import localResolve from 'rollup-plugin-node-resolve'
 import scss from 'rollup-plugin-scss'
 import commonjs from 'rollup-plugin-commonjs'
-import json from 'rollup-plugin-json'
+import css from 'rollup-plugin-css-only'
+import copy from 'rollup-plugin-copy'
+import del from 'rollup-plugin-delete'
 import { eslint } from 'rollup-plugin-eslint'
-import alias from 'rollup-plugin-alias'
 import terser from 'rollup-plugin-terser'
-const path = require('path')
-const cwd = process.cwd()
 
 const isProduction = process.env.NODE_ENV === 'production'
 
 export default {
-	input: 'src/index.js',
-	external: id => {
-		const externals = [
-			'vue',
-		]
-		if (externals.includes(id)) {
-			return true
-		}
-		return false
-	},
+	input: 'src/RichText.vue',
 	output: {
-		file: 'dist/vue-richtext.js',
-		format: 'esm'
+		format: 'esm',
+		file: 'dist/vue-richtext.js'
 	},
+	external: [
+		'unified',
+		'remark-parse',
+		'remark-disable-tokenizers',
+		'remark-external-links',
+		'remark-external-links',
+		'remark-rehype',
+		'rehype-add-classes',
+		'rehype-react',
+		'rehype-sanitize'
```

Comment:
Correct me if I'm wrong but it seems that rehype-sanitize is not used. We probably should either make use of that or add `'html'` as a default to the disableMarkdownBlockFeatures list

Reply:
I cleaned up the dependencies https://github.com/juliushaertl/vue-richtext/pull/226/commits/638a8bd1633530e25089704030e7b533c10a5b36

HTML is disabled by default: https://github.com/juliushaertl/vue-richtext/pull/226/commits/0fe95bb4fd1935e5f59225a05efe293bdcdf8822